dashboard/database.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    connections: {
```

Comment:
Why are there connections for the BMS/RMS in the database.js file?

Reply:
I wasn't sure how connections were handled. I put them there knowing that it's easy enough to take out if needed.